spec/features/versions_spec.rb

```diff
@@ -218,12 +218,21 @@
   scenario 'display changes in splashpages', feature: true, versioning: true, js: true do
     visit admin_conference_splashpage_path(conference.short_title)
     click_link 'Create Splashpage'
+    uncheck('Display tracks')
```

Comment:
Currently when the elements were not selected by default we were updating the _Make splash page public, Display tracks on the splashpage? and Display the registration period on the splashpage?_  but now as all the components are selected by default so by unchecking them in the edit link we are updating all of them except for the _tracks and registration component_ so we have to change the version_spec.rb:237 expect(page).to